flexiblepricing/models.py

```diff
@@ -46,6 +51,28 @@ def __str__(self):
         )
 
 
+class FlexiblePriceTier(TimestampedModel):
+    """
+    The tiers for discounted pricing
+    """
+
+    discount = models.ForeignKey(
+        Discount,
+        null=False,
+        related_name="flexible_price_tiers",
+        on_delete=models.PROTECT,
+    )
+    courseware_content_type = models.ForeignKey(
+        ContentType, on_delete=models.CASCADE
+    )  # add a filter to limit this to program/course contenttypes
```

Comment:
A filter here would be very handy (since now it gives you a list of all models). We do this in the ecommerce Product model - can you cross port that over?

Reply:
Sure, thanks